awx/main/migrations/0108_v370_webhook_mixin_gitea.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.8 on 2020-02-07 00:41
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0107_v370_workflow_convergence_api_toggle'),
```

Comment:
This file likely needs to be updated to reflect migrations that have landed since it was opened:

https://github.com/ansible/awx/tree/devel/awx/main/migrations

Reply:
My migrations fixup script: https://gist.github.com/jbradberry/ab47ba6f8567473f06bd4015020c3ef5

Reply:
I'm happy to update this, but frankly, the process for doing so is a bit confusing to me.  I have my devel branch that this PR is based off of.  I'm not sure what particular incantations of git & django commands are required to fix the migration, and re-push the PR.

Reply:
Ok, I did the rebase and fixup of the migrations, and pushed the resulting branch as https://github.com/jbradberry/awx/tree/gitea-webhook-support.